### ISSUE TITLE
Duplicate repo?

### ISSUE THREAD

**Issue by mathiasbynens** [oligot/awesome-pwa#1]
Have you seen @hemanth’s https://github.com/hemanth/awesome-pwa? Probably better to join forces if possible :)

**Comment by oligot** [comment]
Indeed.
We started a discussion here:
https://github.com/hemanth/awesome-pwa/issues/10
@hemanth: shall I open a PR with the content of this repo ?

**Comment by hemanth** [comment]
@oligot Yes please. I think we can re-org that repo, to list all the apps with their favicons ;) and also having the reading material, video, tuts and stuff? 

@mathiasbynens Thanks for pitching in, the original intent of the repo was to collect all PWA under one hood, but now I am re-thinking 😇

**Comment by oligot** [comment]
PR created: https://github.com/hemanth/awesome-pwa/pull/12

**Comment by hemanth** [comment]
Merged! Thanks.
